### PR TITLE
#2107 Add input field component unit tests

### PR DESCRIPTION
**Pull request by DavidMi-1**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2107)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/common/Mui/Form/__tests__/InputField.test.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import * as generate from 'utils/generate';
+import InputField from '../InputField';
+
+const setup = (override = {}) => {
+  return {
+    input: {
+      name: generate.name(),
+      onChange: jest.fn(),
+      value: generate.name(),
+      //checked: true,
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Mui/Form/__tests__/InputField.test.js]: This should be removed

File: ohara-manager/client/src/components/common/Mui/Form/__tests__/InputField.test.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    meta: {
+      touched: true,
+      error: generate.message(),
+    },
+    testId: 'inputField',
+    ...override,
+  };
+};
+//helperText: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+//errorMessage: PropTypes.string,
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Mui/Form/__tests__/InputField.test.js]: Same here

File: ohara-manager/client/src/components/common/Mui/Form/__tests__/InputField.test.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+describe('<InputField />', () => {
+  afterEach(() => {
+    cleanup();
+    jest.clearAllMocks();
+  });
+
+  it('renders inputField', () => {
+    const props = setup();
+    render(<InputField {...props} />);
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Mui/Form/__tests__/InputField.test.js]: You can also make this a one-liner like:

```suggestion
    render(<InputField {...setup()} />);
```

File: ohara-manager/client/src/components/common/Mui/Form/__tests__/InputField.test.js (hunk 1 above)
Comment (DavidMi-1) [reply ohara-manager/client/src/components/common/Mui/Form/__tests__/InputField.test.js]: ok

File: ohara-manager/client/src/components/common/Mui/Form/__tests__/InputField.test.js (hunk 2 above)
Comment (DavidMi-1) [reply ohara-manager/client/src/components/common/Mui/Form/__tests__/InputField.test.js]: ok

File: ohara-manager/client/src/components/common/Mui/Form/__tests__/InputField.test.js (hunk 3 above)
Comment (DavidMi-1) [reply ohara-manager/client/src/components/common/Mui/Form/__tests__/InputField.test.js]: Wow ! Got it.

**Comment by DavidMi-1** [comment]
@konekoya please review, thanks :)

File: ohara-manager/client/src/components/common/Mui/Form/InputField.js [hunk 4]
```diff
@@ -43,12 +43,14 @@ const InputField = props => {
     width = '100%',
     helperText,
     errorMessage,
+    testId,
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Mui/Form/InputField.js]: I think we should add a default `testId` here? So if the consumer of the component is not supplying a `testId` it won't display an `undefined` value here

File: ohara-manager/client/src/components/common/Mui/Form/__tests__/InputField.test.js [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+const setup = (override = {}) => {
+  return {
+    input: {
+      name: generate.name(),
+      onChange: jest.fn(),
+      value: generate.name(),
+    },
+    meta: {
+      touched: true,
+      error: generate.message(),
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Mui/Form/__tests__/InputField.test.js]: I didn't run your PR on my local. But I'm thinking that maybe we should add a test case which covers the error case?

Comment (DavidMi-1) [reply ohara-manager/client/src/components/common/Mui/Form/__tests__/InputField.test.js]: ok

File: ohara-manager/client/src/components/common/Mui/Form/InputField.js (hunk 4 above)
Comment (DavidMi-1) [reply ohara-manager/client/src/components/common/Mui/Form/InputField.js]: ok

**Comment by DavidMi-1** [comment]
retry manager

File: ohara-manager/client/src/components/common/Mui/Form/InputField.js [hunk 6]
```diff
@@ -43,12 +43,14 @@ const InputField = props => {
     width = '100%',
     helperText,
     errorMessage,
+    testId = 'inputField',
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Mui/Form/InputField.js]: The naming convention for `data-testid` should be `input-field` :)

Comment (DavidMi-1) [reply ohara-manager/client/src/components/common/Mui/Form/InputField.js]: ok

**Comment by DavidMi-1** [comment]
> Looks to me now, Thanks for the PR @DavidMi-1 :)

No, you are not a beauty (woman). I won't look to you XD~
Thanks for review :)

**Comment by konekoya** [comment]
Oops, Looks *GOOD to me now...

**Pull request merged**